README.md

```diff
@@ -172,7 +172,9 @@ in the above example.
 Once your public key is successfully registered, provide vehicle owners with a
 link to `https://tesla.com/_ak/<your_domain_name>`. For example, if you
 registered `example.com`, provide a link to
-`https://tesla.com/_ak/example.com`. The Tesla app will handle the rest.
+`https://tesla.com/_ak/example.com`. The official Tesla Iphone or Android mobile App(version 4.25.5 or above)
```

Comment:
- iPhone
- Space after App
- Lower-case App
- Change 1 to "one"
- "Customers with more than one Tesla product must select the desired vehicle before clicking the link or scanning the QR code."